core/admin/mailu/translations/de/LC_MESSAGES/messages.po

```diff
@@ -53,7 +53,7 @@ msgstr "Client Einrichtung"
 
 #: mailu/sso/templates/sidebar_sso.html:16 mailu/ui/templates/sidebar.html:114
 msgid "Website"
-msgstr "Website"
+msgstr "mailu.io"
```

Comment:
Why not leave this as it is?

Reply:
Yes, it's probably better to just write "Webseite" here. After all mailu.io can be configured AFAIK.

Reply:
changed back